src/Sarif.Sarifer/FileSystem.cs

```diff
@@ -0,0 +1,52 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Generic;
+using System.IO;
+
+namespace Microsoft.CodeAnalysis.Sarif.Sarifer
+{
+    public class FileSystem : IFileSystem
+    {
+        public static readonly FileSystem Instance = new FileSystem();
+
+        /// <summary>
```

Comment:
>/// [](start = 8, length = 3)

FYI you can use `/// <inheritdoc />` rather than repeating the documentation from the interface. I learned that from Gabe a few days ago.

Reply:
Strange that VS didn't suggest that :(
I will update it! :)